easy-coding-standard.neon

```diff
@@ -0,0 +1,10 @@
+# this will be placed in doctrine/coding-stanard, easy-accessible for anyone
```

Comment:
Using the `.neon` syntax is something I really don't understand at all. IMO another reason to use PHPCS here...